testing/trino-product-tests/src/main/java/io/trino/tests/product/utils/QueryExecutors.java

```diff
@@ -155,5 +155,32 @@ public void close()
         };
     }
 
+    public static QueryExecutor onHudi()
+    {
+        return new QueryExecutor()
+        {
+            private final QueryExecutor delegate = testContext().getDependency(QueryExecutor.class, "hudi");
+
+            @Override
+            public QueryResult executeQuery(String sql, QueryParam... params)
+                    throws QueryExecutionException
+            {
+                return delegate.executeQuery(sql, params);
```

Comment:
Note that a retry policy (as it can be seen for the other runners) is currently missing.
In case there will be identified cases worthy to apply a retry policy, then this call can be made with `FailSafe`.